packages/zoltan2/src/problems/Zoltan2_Problem.hpp

```diff
@@ -235,7 +235,7 @@ class Problem : public ProblemRoot {
 };
 
 template <typename Adapter>
-  void Problem<Adapter>::setupProblemEnvironment(ParameterList *params)
+  void Problem<Adapter>::setupProblemEnvironment(ParameterList * /* params */)
```

Comment:
@kddevin should look more closely at this one; perhaps the argument isn't needed.